### PR TITLE
Bootstrap WriteToCSV callbacks

### PR DESCRIPTION
**Pull request by ratuspro**
Create an exporter using Callbacks that writes the results of the expansion to a csv file

### CODE REVIEW

**Review by RicardoEPRodrigues — CHANGES_REQUESTED** [review]
The utils package is missing in `setup.py`.

Everything looks good elsewhere.

**Comment by ratuspro** [comment]
Setup.py now includes `sciencer.utils`.

File: sciencer/utils/csv_callback.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    row = {}
+    row['title'] = paper.title if paper.title is not None else ""
+    row['year'] = str(paper.year) if paper.year is not None else ""
+    row['authors_s2_id'] =  "["+''.join([f"{author_id};" for author_id in paper.authors])[:-1]+"]"
+    return row
+
+field_names = ['title', 'authors_s2_id', 'year', 'accepted']
+
+class WriteToCSVCallbacks(Callbacks):
+    """Class that writes to a csv file
```
Comment (RicardoEPRodrigues) [inline sciencer/utils/csv_callback.py:28]: I would suggest a comment on how the CSV would be structured. But this is optional.

**Review by RicardoEPRodrigues — APPROVED** [review]
All good. Just an optional suggestion.

**Pull request merged**